holidayextras/ui-toolkit
# build all of dist on install

#### What does this PR do? (please provide any background)
builds the distributed version of ui-toolkit on install, instead of having to build it and then commit that
#### What tests does this PR have?
no tests!
#### How can this be tested?
`npm install`, see it creates the same files in this branch that it does when you run `grunt build`
#### Screenshots / Screencast
#### What gif best describes how you feel about this work?
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
